Dockerfile

```diff
@@ -53,9 +53,8 @@ RUN git clone -b $RAFT_VER https://github.com/canonical/raft.git && \
 
 # --- Build dqlite
 
-RUN git clone --depth 1 https://github.com/canonical/dqlite.git && \
```

Comment:
Why include an unrelated change to the build process? 

This breaks if the DQLITE_VER is a commit hash.

Reply:
Sorry, not important, this is okay for now, with `--depth 1` hashes would not work anyways, we may need to support hashes soon tho.